src/test/client.rs

```diff
@@ -500,3 +507,21 @@ async fn saslprep() {
     auth_test_uri("%E2%85%A8", "IV", None, true).await;
     auth_test_uri("%E2%85%A8", "I%C2%ADV", None, true).await;
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn watch_test() {
+    let client = TestClient::new().await;
+    if client.is_standalone() {
+        return;
+    }
+    let pipeline = vec![doc! { "$match": { "x": 3 }}];
+    let options = ChangeStreamOptions::builder()
+        .selection_criteria(SelectionCriteria::ReadPreference(ReadPreference::Primary))
+        .max_await_time(Duration::from_millis(5))
+        .batch_size(10)
+        .full_document(FullDocumentType::UpdateLookup)
+        .build();
+    let result = client.watch(pipeline, Some(options)).await;
+    assert!(result.is_ok());
```

Comment:
This test seems to be failing in CI, but it's hard to tell while due to the fact that this assertion only prints that `result.is_ok()` isn't true. Please update the above call to be `client.watch(pipeline, options).await.unwrap()` so that the full error will be printed in the case of a failure.

Reply:
fixed